server/job-workers/job-worker-server.cpp

```diff
@@ -175,7 +175,7 @@ void JobWorkerServer::init() {
     clear_event(result_pipe[0]);
   }
 
-  read_job_connection = epoll_insert_pipe(pipe_for_read, read_job_fd, &php_jobs_server, nullptr);
+  read_job_connection = epoll_insert_pipe(pipe_for_read, read_job_fd, &php_jobs_server, nullptr, EVT_LEVEL);
```

Comment:
Please, add the following test case:
setup: 2 job workers + N http workers (may be 4, to be sure that job messages would be enough)
use instance_cache for synchronization

Http worker:
1) starts 2 sync jobs
2) reads in while from instance_cache to be sure, that jobs are started 
3) starts N (>=2, 3 is fine) simple jobs
4) writes something into instance_cache (any special key), so sync job are going to finish
5) wait simple jobs reply
6) wait sync jobs reply

Job worker (SYNC):
1) writes something into isntance_cache (any special key)
2) read in while from instance_cache to be sure, that http worker has started simple jobs
3) finish

Job worker (simple): up to you, x^2 would be enough